pkg/cmd/server/api/v1/conversions.go

```diff
@@ -90,14 +90,23 @@ func SetDefaults_MasterConfig(obj *MasterConfig) {
 			obj.NetworkConfig.ServiceNetworkCIDR = "10.0.0.0/24"
 		}
 	}
+	if len(obj.NetworkConfig.ClusterNetworks) == 0 {
+		obj.NetworkConfig.ClusterNetworks = []ClusterNetworkEntry{{CIDR: obj.NetworkConfig.DeprecatedClusterNetworkCIDR, HostSubnetLength: obj.NetworkConfig.DeprecatedHostSubnetLength}}
+	}
```

Comment:
When both `ClusterNetworks` and `DeprecatedClusterNetworkCIDR/DeprecatedHostSubnetLength` are defined, it would be nice if can we log a warning. Something like 'Ignoring clusterNetworkCIDR/hostSubnetLength fields in favor of clusterNetwork'.